x-pack/plugins/observability_solution/apm/public/components/app/settings/agent_configurations/list/confirm_delete_modal.tsx

```diff
@@ -66,11 +66,9 @@ async function deleteConfig(config: Config, toasts: NotificationsStart['toasts']
     await callApmApi('DELETE /api/apm/settings/agent-configuration 2023-10-31', {
       signal: null,
       params: {
-        body: {
-          service: {
-            name: config.service.name,
-            environment: config.service.environment,
-          },
+        query: {
```

Comment:
@bryce-b this is a public API, I'm not sure if we can just change its signature. I think the APM agents team are the ones using it, have you checked with them?

Reply:
Would it be better if I create a new API instead? If they do indeed use it, there won't be a clean way to make this change and continue supporting old agents.

Reply:
I suggest keeping the same API but implementing both query and body parameters. With query params overwriting body. That way, we stay backward compatible and can provide a working API for public (we might document only query parameters for simplicity). What do you think? @bryce-b @cauemarcondes

Reply:
We have found the root cause of this problem. Lets close this PR, please.